spec/lib/second_factor/actions/grant_admin_spec.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe SecondFactor::Actions::GrantAdmin do
+  fab!(:admin) { Fabricate(:admin) }
+  fab!(:user) { Fabricate(:user) }
+
+  before do
+    AdminConfirmation.cleanup_redis
```

Comment:
I think we always want to be cleaning up after and not before. If you need to clean up before a test, it means that we're already leaking state across tests which can make our test flaky.